yarn-project/prover-client/src/prover/bb_prover.ts

```diff
@@ -46,14 +48,18 @@ export type BBProverConfig = {
   bbWorkingDirectory: string;
   acvmBinaryPath: string;
   acvmWorkingDirectory: string;
+  // list of circuits supported by this prover. defaults to all circuits if empty
+  circuitFilter?: ServerProtocolArtifact[];
 };
 
 /**
  * Prover implementation that uses barretenberg native proving
  */
 export class BBNativeRollupProver implements CircuitProver {
-  private verificationKeyDirectories: Map<ServerProtocolArtifact, string> = new Map<ServerProtocolArtifact, string>();
-  constructor(private config: BBProverConfig) {}
+  constructor(
+    private config: BBProverConfig,
```

Comment:
```suggestion
  private constructor(
```

Reply:
Yes. Will update in my next PR as I'm changing things agian.